### ISSUE TITLE
Jupyter kernel crashes on running injections module in Mac

### ISSUE THREAD

**Issue by jsarneja** [whylabs/langkit#225]
I am running an injections module example and my jupyter kernel keeps dying.

System: Macbook pro 32GB Intel chip
Python version: 3.9.18

**Steps to recreate**:
In mac terminal run the following code for creating new conda environment:
```console
conda create -n jailbreak_test_env python=3.9
conda activate jailbreak_test_env
pip install langkit[all]==0.0.28
pip install notebook
jupyter notebook
```
In jupyter run the code:
```python
from langkit import injections, extract
prompt = "Tell me a joke."
result = extract({"prompt":prompt})
print(f"Prompt: {result['prompt']}\nInjection score: {result['prompt.injection']}")
```
![image](https://github.com/whylabs/langkit/assets/42511082/76d4361b-47fd-4e0b-9c4b-cf12d52bf816)

On running the code in terminal, the following error is displayed:
![image](https://github.com/whylabs/langkit/assets/42511082/b995cf3f-b1c8-4c34-9067-bce69659d5dd)

My guess is this issue might be related to #161 and #162? Not sure if it was fixed then..

**Comment by FelipeAdachi** [comment]
Hello @jsarneja ,

This issue should be fixed on the latest release, 0.0.30.

Could you test it to make sure it solves your problem?
Thank you!

**Comment by FelipeAdachi** [comment]
As no further comments were added, I'm closing this issue.
